a5/Dockerfile

```diff
@@ -0,0 +1,7 @@
+# rosindustrial/a5:kinetic
+FROM rosindustrial/noether:kinetic
+MAINTAINER Austin.Deric@gmail.com
```

Comment:
`MAINTAINER` is [deprecated](https://docs.docker.com/engine/reference/builder/#maintainer-deprecated)